### PR TITLE
add node to lumen starter container

### PR DESCRIPTION
**Pull request by millidavids**

### CODE REVIEW

**Comment by millidavids** [comment]
This is in preparation for adding in aglio to the container which is installed via npm. @jlaswell @bkuhl

File: infrastructure/fpm/Dockerfile [hunk 1]
```diff
@@ -10,10 +10,14 @@ ADD . /var/www/html
 
 WORKDIR /var/www/html
 
+# Install Node.js
+RUN curl -sL https://deb.nodesource.com/setup | bash - \
+    && apt-get install -y build-essential nodejs \
+    && npm install -g npm
```
Comment (bkuhl) [inline infrastructure/fpm/Dockerfile]: Can you update this comment to reflect this is used by algio to generate documentation?

**Comment by bkuhl** [comment]
LGTM - Assuming comment added

File: infrastructure/fpm/Dockerfile (hunk 1 above)
Comment (millidavids) [inline infrastructure/fpm/Dockerfile]: Absolutely

**Pull request merged**